app/pages/admin/project-status.jsx

```diff
@@ -124,17 +124,23 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  saveProject(project) {
+    return project.save()
+      .catch(error => this.setState({ error }));
+  }
+
   handleFeaturedProjectChange({ target }) {
-    const { featured, project } = this.state;
-    if (featured) {
-      featured.update({ featured: false });
-      featured.save()
-        .then(() => this.setState({ featured }))
+    const { featuredProject, project } = this.state;
+    project.update({ featured: target.checked });
+    if (featuredProject) {
+      featuredProject.update({ featured: false });
+      return this.saveProject(featuredProject)
+        .then(() => this.saveProject(project))
+        .then(() => this.setState({ featuredProject: project }))
         .catch(error => this.setState({ error }));
+    } else {
+      return this.saveProject(project);
```

Comment:
Is this line still needed?

Reply:
It's useful if somehow no featured project is set. The DB allows that.

Reply:
You can get here if a featured project is set in Panoptes, but it hasn't been set in the component state yet. In that case, if it arises, the save here will fail with an error.

Reply:
Sorry I'm not seeing how. This is only called when clicking on the checkbox. By then the `componentDidMount` will have set the state, no? Do you think there's a scenario where it's possible to click on it before that happens?

Reply:
`featuredProject` is initially null, so that's the case I was thinking of. Is it also null after errors? Just reading through the code, it looked like the logic allows you to save here in circumstances where `featuredProject` isn't determined, and you might end up with an uncaught error.